uberenv.py

```diff
@@ -365,7 +365,29 @@ def setup_paths_and_dirs(self):
 
         UberEnv.setup_paths_and_dirs(self)
 
-        self.ports = pjoin(self.uberenv_path, "vcpkg_ports","*")
+        # Find path to vcpkg ports
```

Comment:
It seems like we should have a utility function to get the item from the command line with a fallback to uberenv_config

Reply:
I agree.  We should do a followup PR to do smaller cleanup things like this as well as add some CI checks to PRs.

Reply:
logged #50 so we don't forget.